test/aggregate.spec.js

```diff
@@ -113,58 +119,69 @@ export const testAggregate = {
       size: Aggregate.PaddedSize.from(1 << 20),
     })
 
-    builder.write({
-      size: Piece.PaddedSize.from(131072),
-      link: Link.parse(
-        `baga6ea4seaqievout3bskdb76gzldeidkhxo6z5zjrnl2jruvwfwvr2uvvpuwdi`
-      ),
-    })
+    builder.write(
+      Piece.fromInfo({
+        size: Piece.PaddedSize.from(131072),
+        link: Link.parse(
+          `baga6ea4seaqievout3bskdb76gzldeidkhxo6z5zjrnl2jruvwfwvr2uvvpuwdi`
+        ),
+      })
+    )
 
-    const estimate = builder.estimate({
-      size: Piece.PaddedSize.from(524288),
-      link: Link.parse(
-        `baga6ea4seaqkzsosscjqdegbhqrlequtm7pbjscwpeqwhrd53cxov5td34vfojy`
-      ),
-    })
+    const estimate = builder.estimate(
+      Piece.fromInfo({
+        size: Piece.PaddedSize.from(524288),
+        link: Link.parse(
+          `baga6ea4seaqkzsosscjqdegbhqrlequtm7pbjscwpeqwhrd53cxov5td34vfojy`
+        ),
+      })
+    )
 
     assert.match(estimate.error, /Pieces are too large to fit/)
 
     assert.throws(
       () =>
-        builder.write({
-          size: Piece.PaddedSize.from(524288),
-          link: Link.parse(
-            `baga6ea4seaqkzsosscjqdegbhqrlequtm7pbjscwpeqwhrd53cxov5td34vfojy`
-          ),
-        }),
+        builder.write(
+          Piece.fromInfo({
+            size: Piece.PaddedSize.from(524288),
+            link: Link.parse(
+              `baga6ea4seaqkzsosscjqdegbhqrlequtm7pbjscwpeqwhrd53cxov5td34vfojy`
+            ),
+          })
+        ),
       /Pieces are too large to fit in the index/
     )
   },
 
-  'pass bad value to estimate': async (assert) => {
+  'skip! pass bad value to estimate': async (assert) => {
```

Comment:
what is the reason to skip this one?

Reply:
It became impossible to pass bad values (see line 172) because we switch to height and therefor size is always pow2. I probably will remove this test, I just had it disabled as I was trying to figure out what to do with it.